Resources/views/base.html.twig

```diff
@@ -35,7 +35,11 @@
         <nav class="navbar navbar-static-top">
             <div class="container">
                 <div class="navbar-header">
-                    <a href="{{ path('ticketing_perso') }}" class="navbar-brand"><b></b>My Tickets</a>
+                    {% if app.user %}
+                        <a href="{{ path('ticketing_perso') }}" class="navbar-brand"><b></b>My Tickets</a>
+                    {% else %}
+                        <span class="navbar-brand"><b>My Tickets</b></span>
```

Comment:
Maybe redirect the user to the connection page ?

Reply:
The connect page is on the demo, so I can rollback this change and when the user will go to this page, he will be redirected to the login page